src/sentry/api/event_search.py

```diff
@@ -37,6 +37,8 @@
 # before the asterisk is actually escaping the asterisk.
 WILDCARD_CHARS = re.compile(r"(?<!\\)(\\\\)*\*")
 
+HEXADECIMAL_16_DIGITS = re.compile("^[0-9a-fA-F]{16}$")
```

Comment:
Should I include A-F in the regex? 🤔 The accompanying error message I created doesn't mention A-F so I feel like maybe we should tighten this check.

Reply:
IMO we should include A-F, especially since it matches https://github.com/getsentry/relay/blob/a1303ce8eeb9ff479c91fa6642f197ba4ab2ff70/relay-general/src/protocol/contexts.rs#L314

I think its implied that its case insensitive, but we could try to reword both this and event_Id to include A-F if we get feedback on this